lib/resque/stat.rb

```diff
@@ -7,12 +7,14 @@ module Resque
   #   Kill a stat: Stat.clear(name)
   module Stat
     extend self
-    
-    # Direct access to the Redis instance.
-    def redis
-      Resque.redis
+
+    def data_store
+      @data_store ||= Resque.redis
+    end
+
+    def data_store=(data_store)
+      @data_store = data_store
     end
-    alias :data_store :redis
```

Comment:
Can we keep the alias to `redis` with a deprecation warning?

Reply:
Done https://github.com/resque/resque/pull/1652/commits/7f79b5b17d275fe664f747dc2ddb31994e66b056